app/models/admin/r_and_r_item.rb

```diff
@@ -0,0 +1,100 @@
+class Admin::RAndRItem < ApplicationRecord
+  belongs_to :digitization_queue_item, optional: true
```

Comment:
This allows multiple digitalization queue items for one R&R item. 

If we only want one-to-one, we need to use the rarely used Rails feature `has_one`. 

That means the foreign key could be on RAndRItem OR DigitizationQueueItem -- I'm not sure which is best but you can consider which you want to put it on.

Reply:
Regardless of what the model says though --- you need to check your UI, is the "make digitzation queue item" button still available AFTER you've already done it once. Should it be?

Reply:
I've decided to go with a one-to-many relationship, per our earlier discussion. The button remains available after a new dq item is created.